src/compose.js

```diff
@@ -12,15 +12,18 @@ var reverse = require('./reverse');
  * @memberOf R
  * @since v0.1.0
  * @category Function
- * @sig ((y -> z), (x -> y), ..., (o -> p), ((a, b, ..., n) -> o)) -> ((a, b, ..., n) -> z)
- * @param {...Function} functions
+ * @sig ((x -> z), ..., (b -> c), ((a1, a2, ...) -> b), (a1, a2, ...)) -> ((a1, a2, ...) -> z)
```

Comment:
It does kind of come down to whether we want to follow formal mathematic notation or just make something that looks readable.

I found the original (a, b, ..., n) to be pretty arbitrary, and it implied to me that the input argument types were somehow related to the other function types. Personally I found it clearer to show that only the first function was variadic by using a1, a2, ...

The y-> x switch is a typo, I'll fix that.